mawabot/cogs/general.py

```diff
@@ -68,6 +68,13 @@ def __init__(self, bot):
             rolls = ' + '.join(rolls)
             await ctx.send(content=f'🎲 {rolls} = {total}')
 
+    @commands.command()
+    async def nick(self, ctx):
+        ''' Changes the users nickname '''
+        # I did this to allow nicknames with spaces, change it if you like
+        nickname = " ".join(ctx.message.content.split(" ")[1:])
```

Comment:
Instead of doing this discord.py will put everything in 1 argument if you put * before the argument `aysnc def nick(self, ctx, *, nick: str)` will put everything into nick